packages/@lwc/synthetic-shadow/src/shared/static-html-collection.ts

```diff
@@ -46,10 +42,7 @@ StaticHTMLCollection.prototype = create(HTMLCollection.prototype, {
         writable: true,
         enumerable: true,
         configurable: true,
-        value(name: string) {
-            if (isValidHTMLCollectionName(name) && this[name]) {
```

Comment:
I don't remember the details of this, but it seems important.

Reply:
Not sure how this was working as expected before for a string type argument. PM's change makes sense.
Question: 
1. What is this comment about looping in reverse in L47?
2. If the existing approach is working as expected, then it seems to be a faster solution compared to looping over the item and getting the `id` and `name` attribute value.